authority/provisioner/wire/wire_options.go

```diff
@@ -0,0 +1,62 @@
+package wire
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+)
+
+// Options holds the Wire ACME extension options
+type Options struct {
+	OIDC *OIDCOptions `json:"oidc,omitempty"`
+	DPOP *DPOPOptions `json:"dpop,omitempty"`
+
+	validateOnce  sync.Once
+	validationErr error
+}
+
+// GetOIDCOptions returns the OIDC options.
+func (o *Options) GetOIDCOptions() *OIDCOptions {
+	if o == nil {
+		return nil
+	}
+	return o.OIDC
+}
+
+// GetDPOPOptions returns the DPoP options.
+func (o *Options) GetDPOPOptions() *DPOPOptions {
+	if o == nil {
+		return nil
+	}
+	return o.DPOP
+}
+
+func (o *Options) Validate() error {
+	o.validateOnce.Do(
```

Comment:
See previous comment. Check if changing the provisioner using the admin API or HUP will reset the full provisioner and, consequently, this once, too.

Reply:
I've removed this for now. I think it would work as expected with the `ReloadAdminResources` doing a new `Init` for every provisioner and replacing the provisioners, but I didn't want to block on this right now. Also, I think it should actually be part of the `Init` (or just after that), instead of it being triggered inside of the actual handler, so let's do it the right way (later), instead of doing it now